shardingsphere-governance/shardingsphere-governance-core/shardingsphere-governance-core-common/src/main/java/org/apache/shardingsphere/governance/core/yaml/swapper/RuleSchemaMetaDataYamlSwapper.java

```diff
@@ -47,22 +47,26 @@ public YamlRuleSchemaMetaData swapToYamlConfiguration(final RuleSchemaMetaData m
         YamlRuleSchemaMetaData result = new YamlRuleSchemaMetaData();
         result.setConfiguredSchemaMetaData(convertYamlSchema(metaData.getConfiguredSchemaMetaData()));
         Map<String, YamlSchemaMetaData> unconfigured = metaData.getUnconfiguredSchemaMetaDataMap().entrySet().stream()
-                .collect(Collectors.toMap(Entry::getKey, entry -> convertYamlSchema(entry.getValue())));
```

Comment:
Do you think the following change is possible?

```java
//  YamlRuleSchemaMetaData

private Map<String, YamlSchemaMetaData> unconfiguredSchemaMetaDataMap;

to

private Map<String, Collection<String>> unconfiguredSchemaMetaDataMap;
```

Since a similar change is made in `RuleSchemaMetaData`, I suppose it will be better to keep `YamlRuleSchemaMetaData `  and `RuleSchemaMetaData` same. 

What do you think? FYI, please note this change will also alter the content for the registry center (`SchemaChangedListener` will be involved)

Reply:
Ok, will make YamlRuleSchemaMetaData and RuleSchemaMetaData the same